src/main/java/org/broadinstitute/hellbender/cmdline/argumentcollections/CLPConfigurationArgumentCollection.java

```diff
@@ -0,0 +1,37 @@
+package org.broadinstitute.hellbender.cmdline.argumentcollections;
+
+import htsjdk.samtools.util.Log;
+
+import java.io.File;
+import java.io.Serializable;
+import java.nio.file.Path;
+import java.util.List;
+
+/**
+ * Arguments used to configure the {@link org.broadinstitute.hellbender.cmdline.CommandLineProgram}.
+ *
+ * <p>The purpose of this abstract class is to allow downstream projects to decide which configuration arguments they
+ * want to expose to the CLP.
+ *
+ * <p>In addition, this class might contain arguments for documentation-only (e.g., arguments from a wrapper script).
+ *
+ * @author Daniel Gomez-Sanchez (magicDGS)
+ */
+public abstract class CLPConfigurationArgumentCollection implements Serializable {
+    private static final long serialVersionUID = 1L;
+
+    /**
+     * Return a non-empty list of temp directories.
+     */
+    public abstract List<Path> getResolvedTmpDirectories();
```

Comment:
I would just call this `getTmpDirectories`.

Reply:
Done.